net/http/CMakeLists.txt

```diff
@@ -58,3 +58,16 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RHTTP
                               LIBRARIES ${SYSLIBS} ZLIB::ZLIB ${RT_LIBRARIES} ${FASTCGI_LIBRARY}
                                         ${OPENSSL_LIBRARIES} ${CMAKE_DL_LIBS}
                               DEPENDENCIES Thread RIO)
+
+target_compile_definitions(RHTTP PUBLIC -DUSE_WEBSOCKET)
+
+if(ssl)
+   target_compile_definitions(RHTTP PUBLIC -DNO_SSL_DL)
+   if (_ssl11)
+      target_compile_definitions(RHTTP PUBLIC -DOPENSSL_API_1_1)
```

Comment:
Also in response to Guilherme: civetweb headers are only used by ROOT internally. As long as all those parts (RHTTP and civetweb itself) share the same definition of `OPENSSL_API_1_1` we're good.

Reply:
Strange, while I see message during compilation:

    -- Use SSL API VERSION 1.1 for civetweb

And value for me is `"1.1.0h"`. But true - value "1.1.0.1" does not work. Have no idea